src/pages/List/List_Products.js

```diff
@@ -17,13 +18,20 @@ function ListProducts() {
   useEffect(() => {
     if (token.current) {
       /* Get items */
-      const unsubscribe = onSnapshot(getListFromDB(token.current), (doc) => {
-        list.current = doc.data();
-        setItems(doc.data().items);
-      });
-      return () => {
-        unsubscribe();
+
+      const getItemsFromList = async () => {
+        const docRef = getListFromDB(token.current);
+        const docSnap = await getDoc(docRef);
+        if (docSnap.exists()) {
+          const products = docSnap.data().items;
+          if (products !== undefined) {
```

Comment:
maybe avoid having two `return` statements?
```suggestion
          if (products === undefined) {
            return [];
          }
          setItems(products);
```

Reply:
Ya quedó... Sin ninguno de los dos 😝